src/Searching.ts

```diff
@@ -605,7 +605,7 @@ function eventIndexSearch(
     let searchPromise: Promise<ISearchResults>;
 
     if (roomId !== undefined) {
-        if (client.isRoomEncrypted(roomId)) {
+        if (client.getCrypto()?.isEncryptionEnabledInRoom(roomId)) {
```

Comment:
this if statement will always be truthy as a Promise is truthy, it should be awaited